src/main/java/DoTask.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.List;
+class DoTask extends Event {
+    boolean firstGreet;
+    String lastCommand;
+    TaskList taskList;
```

Comment:
removedTask look like an action, why you choose to store it as a string datatype?